src/infra/messaging-outbox/repo-tests/src/outbox_message_repository_test_suite.rs

```diff
@@ -341,6 +347,50 @@ pub async fn test_reading_messages_above_max_with_multiple_producers(catalog: &C
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+pub async fn test_outbox_messages_version(catalog: &Catalog) {
```

Comment:
Q: Can also add not only an example with the past version, but also the addition of a future version?

Reply:
We suppose we will work only with latest. I can imagine only 1 case when we need to ignore highest versions, it's when we deployed and reverted changes but to test it might be tricky, because once we rich a `higher mock version` we should always go back and change it.